packages/gatsby-theme-carbon/src/components/Aside/Aside.js

```diff
@@ -11,15 +11,41 @@ export default class Aside extends React.Component {
      * Specify a custom class
      */
     className: PropTypes.string,
+
+    /**
+     * Specify an `aria-label` value to provide a label to the inner aside element.
+     */
+    ariaLabel: PropTypes.string,
+
+    /**
+     * Specify an `aria-labelledby` value that references the id of an existing
+     * element to serve as a label for the inner aside element.
+     */
+    ariaLabelledBy: PropTypes.string,
   };
 
   render() {
-    const { children, className } = this.props;
+    const {
+      ariaLabel,
```

Comment:
Since we're not using a using a div, we don't have to de-structure these or switch the case, we can just pass them through right?

That way dev's use them just like they would on a native `aside` jsx element.